content/Education and Kits/Science Journal App/Upload-the-Science-Journal-firmware.md

```diff
@@ -48,8 +53,9 @@ When the upload is completed you can [connect your board to the Science Journal
 
 You need to install the following libraries:
 
-- `Arduino_ScienceJournal`
-- `Adafruit LSM9DS1 Library` (MKR WiFi 1010 only)
+- `Arduino_ScienceKitCarrier` (If using the Nano RP2040 Connect)
+- `Arduino_ScienceJournal`    (If using the Nano 33 BLE Sense or MKR WiFi 1010)
+- `Adafruit LSM9DS1 Library` (If using the MKR WiFi 1010)
```

Comment:
There used to be an issue where you had to install this library since it wasn't listed as a dependency. This appears to [have been fixed](https://github.com/arduino-libraries/Arduino_ScienceJournal/blob/master/library.properties) in the version now recommended for MKR WiFi 1010, so it should no longer be required.
```suggestion
```